app/mobile-content/exhibits/edit-exhibit/edit-exhibit.component.ts

```diff
@@ -150,6 +176,7 @@ export class EditExhibitComponent implements OnInit {
       (selectedMedium: Medium) => {
         if (selectedMedium) {
           this.exhibit.image = selectedMedium.id;
+          this.previewImage(selectedMedium.id);
```

Comment:
I think it would be cleaner, to have something like `this.url = previewImage(selectedMedium.id)`. It simply doesn't look too nice, having two assignments and in-between a function call with side-effects.